docs/model_library/operational_water_management/index.rst

```diff
@@ -593,42 +600,74 @@ Flow is only allowed in a given direction if the binary indicator for that direc
-
 .. math::
 
-    \textcolor{red}{F_{l,l,t}^{Piped}}\leq \textcolor{red}{y_{l,l,t}^{Flow}} \cdot \textcolor{green}{M^{Flow}}
+    \textcolor{red}{F_{l,\tilde{l},t}^{Piped}} \leq \textcolor{red}{y_{l,\tilde{l},t}^{Flow}} \cdot \textcolor{green}{M^{Flow}}
 
 
 
 **Storage Site Balance:** :math:`\forall s \in S, t \in T`
 
 For each storage site and for each time period, if it is the first time period, the storage level is the initial storage. Otherwise, the storage level is equal to the storage level in the previous time period plus water inputs minus water outputs.
 
-.. math::
+For :math:`t = 1`:
 
-    \textcolor{red}{L_{s,t}^{Storage}} = \textcolor{green}{\lambda_{s,t=1}^{Storage}}+\textcolor{red}{L_{s,t-1}^{Storage}}+\sum\nolimits_{(n,s)\in NSA}\textcolor{red}{F_{l,l,t}^{Piped}} +\sum\nolimits_{(p,s)\in PST}\textcolor{red}{F_{l,l,t}^{Trucked}}
+.. math::
 
-        +\sum\nolimits_{(p,s)\in CST}\textcolor{red}{F_{l,l,t}^{Trucked}}-\sum\nolimits_{(s,n)\in SNA}\textcolor{red}{F_{l,l,t}^{Piped}}-\sum\nolimits_{(s,p)\in SCA}\textcolor{red}{F_{l,l,t}^{Piped}}-\sum\nolimits_{(s,k)\in SKA}\textcolor{red}{F_{l,l,t}^{Piped}}
+    \textcolor{red}{L_{s,t}^{Storage}}
+        = \textcolor{green}{\lambda_{s,t=1}^{Storage}}
+        + \sum_{l \in L | (l, s) \in LLA}\textcolor{red}{F_{l,s,t}^{Piped}}
+        + \sum_{l \in L | (l, s) \in LLT}\textcolor{red}{F_{l,s,t}^{Trucked}}
+        - \sum_{l \in L | (s, l) \in LLA}\textcolor{red}{F_{s,l,t}^{Piped}}
+        - \sum_{l \in L | (s, l) \in LLT}\textcolor{red}{F_{s,l,t}^{Trucked}}
 
-        -\sum\nolimits_{(s,r)\in SRA}\textcolor{red}{F_{l,l,t}^{Piped}}-\sum\nolimits_{(s,o)\in SOA}\textcolor{red}{F_{l,l,t}^{Piped}}-\sum\nolimits_{(s,p)\in SCT}\textcolor{red}{F_{l,l,t}^{Trucked}}-\sum\nolimits_{(s,k)\in SKT}\textcolor{red}{F_{l,l,t}^{Trucked}}
+For :math:`t > 1`:
 
+.. math::
 
+    \textcolor{red}{L_{s,t}^{Storage}}
+        = \textcolor{red}{L_{s,t-1}^{Storage}}
+        + \sum_{l \in L | (l, s) \in LLA}\textcolor{red}{F_{l,s,t}^{Piped}}
+        + \sum_{l \in L | (l, s) \in LLT}\textcolor{red}{F_{l,s,t}^{Trucked}}
+        - \sum_{l \in L | (s, l) \in LLA}\textcolor{red}{F_{s,l,t}^{Piped}}
+        - \sum_{l \in L | (s, l) \in LLT}\textcolor{red}{F_{s,l,t}^{Trucked}}
 
 **Pipeline Capacity:**
 
-:math:`\forall (l,l) \in {PCA,PNA,PPA,CNA,NNA,NCA,NKA,NSA,NRA,\ldots ,SOA}, [t \in T]`
+:math:`\forall (l,l) \in LLA, [t \in T]`
 
 .. math::
 
     \textcolor{red}{F_{l,l,[t]}^{Capacity}} = \textcolor{green}{\sigma_{l,l}^{Pipeline}}+\textcolor{red}{S_{l,l}^{PipelineCapacity}}
 
-:math:`\forall (l,l) \in {PCA,PNA,PPA,CNA,NNA,NCA,NKA,NSA,NRA,\ldots ,SOA}, t \in T`
+:math:`\forall (l,l) \in LLA, t \in T`
 
 .. math::
 
     \textcolor{red}{F_{l,l,t}^{Piped}}\leq \textcolor{red}{F_{l,l,[t]}^{Capacity}}
 
+**Pipeline Capacity Construction/Expansion:** :math:`\forall \textcolor{blue}{(l,\tilde{l}) \in LLA}, [\textcolor{blue}{t \in T}]`
```

Comment:
```suggestion
**Pipeline Capacity Construction/Expansion:** :math:`\forall (l,\tilde{l}) \in LLA, [t \in T]`
```
Remove blue text to remain consistent with the rest of the operational model documentation.